apps/aecore/lib/aecore/contract/contract.ex

```diff
@@ -26,9 +32,48 @@ defmodule Aecore.Contract.Contract do
         }
 
   defstruct [:id, :owner, :vm_version, :code, :store, :log, :active, :referers, :deposit]
-
+  use ExConstructor
```

Comment:
Do you need `use ExConstructor`?

Reply:
Isn't it needed for all Structures? This is a structure.

Reply:
no, we probably won't need it here, as the `create` is implemented manually